Capgemini.Xrm.CdsDataMigrator/Capgemini.Xrm.CdsDataMigratorLibrary/UserControls/ImportPage.cs

```diff
@@ -118,6 +114,24 @@ string IImportPageView.AskForFilePathToSave(string existingFileName)
             return saveFileDialog.FileName;
         }
 
+        [ExcludeFromCodeCoverage]
+        public void SetMetadataService(IMetadataService metaDataService)
+        {
+            importLookupMappingsFormPresenter.MetaDataService = metaDataService;
+        }
+
+        [ExcludeFromCodeCoverage]
+        public void SetOrganizationService(IOrganizationService organizationService)
+        {
+            importLookupMappingsFormPresenter.OrganizationService = organizationService;
+        }
+
+        [ExcludeFromCodeCoverage]
+        public void SetViewHelpers(IViewHelpers viewHelpers)
+        {
+            importLookupMappingsFormPresenter.ViewHelpers = viewHelpers;
+        }
```

Comment:
What are your thoughts on merging these into `SetServices(...)`?

Reply:
@tdashworth - Yeah makes sense. And include SetViewHelpers in this?